rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -94,11 +96,90 @@ object Jupyter {
   def whiskers[K, N](samples: Seq[Map[String, Double]]): Plot = {
```

Comment:
This should use `Seq[Map[K, N]]` to match the other signature right?

Reply:
see previous comment about precis